src/test/kotlin/RequestParserTest.kt

```diff
@@ -0,0 +1,41 @@
+import kotlin.test.*
+import request.RequestParser
+import request.Request
+
+class RequestParserTest {
+    @Test fun testReturnRequestWhenSimpleGetFromClient() {
```

Comment:
I just noticed that you have the `@Test` annotation on the same line as the `fun` declaration. In Java, I'm accustomed to seeing annotations on their own line(s) before the method declaration. Is the style that you're using here idiomatic in Kotlin?

Reply:
From my understanding thus far, yes. `kotlin.test` library provides the same annotations.

Reply:
Apologies, I didn't answer your question correctly! After doing some further reading I believe you are correct that the `@Test` annotation should be on its own line. Since I am using the JUnit platform for testing, according to this [resource](https://kotlinlang.org/docs/jvm-test-using-junit.html#add-the-code-to-test-it) I found on the Kotlin website, the annotation should be on its own line.